scripts/ubuntu/Readme.md

```diff
@@ -0,0 +1,66 @@
+# This is an easy scripts to run samples from scratch for Ubuntu users.
+
+
+# Steps
+## Clone the OpenVSLAM
+Probablly you already run git clone, but I redisplay the procedure to show you complete steps.
```

Comment:
Please add a indentation.
https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines

```suggestion
# Steps

## Clone the OpenVSLAM

Probablly you already run git clone, but I redisplay the procedure to show you complete steps.
```

Please correct other similar sentences as well.